etcdmgmt/etcd-mgmt.go

```diff
@@ -79,3 +59,44 @@ func StartEtcd() (*exec.Cmd, error) {
 
 	return etcdCmd, nil
 }
+
+func StartInitialEtcd() (*exec.Cmd, error) {
+	hostname, err := os.Hostname()
+	if err != nil {
+		log.Fatal("Could not able to get hostname")
+		return nil, err
+	}
+
+	listenClientUrls := "http://" + hostname + ":2379"
+	advClientUrls := "http://" + hostname + ":2379"
+	listenPeerUrls := "http://" + hostname + ":2380"
+	initialAdvPeerUrls := "http://" + hostname + ":2380"
+
+	args := []string{"-listen-client-urls", listenClientUrls,
+		"-advertise-client-urls", advClientUrls,
+		"-listen-peer-urls", listenPeerUrls,
+		"-initial-advertise-peer-urls", initialAdvPeerUrls,
+		"--initial-cluster", "default=" + listenPeerUrls}
+
+	return StartEtcd(args)
+}
+
+func ReStartEtcd() (*exec.Cmd, error) {
+	hostname, err := os.Hostname()
+	if err != nil {
+		log.Fatal("Could not able to get hostname")
+		return nil, err
+	}
+
+	listenClientUrls := "http://" + hostname + ":2379"
+	advClientUrls := "http://" + hostname + ":2379"
+	listenPeerUrls := "http://" + hostname + ":2380"
+	initialAdvPeerUrls := "http://" + hostname + ":2380"
+
+	args := []string{"-listen-client-urls", listenClientUrls,
+		"-advertise-client-urls", advClientUrls,
+		"-listen-peer-urls", listenPeerUrls,
+		"-initial-advertise-peer-urls", initialAdvPeerUrls}
+
+	return StartEtcd(args)
+}
```

Comment:
Looking at these three flavours I firmly believe the functions can be still made one with varying args.